<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>805</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>PAD System User Interface - Â© 2019 Massacusetts General Hospital, Center for Systems Biology (http://csb.mgh.harvard.edu)</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>400</height>
       </size>
      </property>
      <property name="title">
       <string>Experimental Parameters</string>
      </property>
      <widget class="QLabel" name="labelIntervalSlider_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>220</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Samples</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelExpName">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>23</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Experiment Name: </string>
       </property>
      </widget>
      <widget class="QLabel" name="labelSampleRate">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>148</y>
         <width>72</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sampling Rate:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNumSamples">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>220</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSlider" name="sliderInterval">
       <property name="geometry">
        <rect>
         <x>114</x>
         <y>55</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="labelIntervalSlider_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>second(s)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelIntervalMinutes">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelDurationSeconds">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>60</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>260</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X hours, X minutes</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelSampleRate_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>72</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of Samples</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelSampleRateMillis">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSlider" name="sliderDuration">
       <property name="geometry">
        <rect>
         <x>114</x>
         <y>104</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <number>120</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelDuration">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>104</y>
         <width>57</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sample For:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelSampleRate_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Total Run Time:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelIntervalSlider_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>170</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>millisecond(s)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelIntervalSlider">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>minute(s)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSlider" name="sliderSampleRate">
       <property name="geometry">
        <rect>
         <x>114</x>
         <y>148</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>100</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelInterval">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>55</y>
         <width>72</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sample Every: </string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonStartDiscrete">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>440</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Discrete
Experiment</string>
       </property>
      </widget>
      <widget class="QSlider" name="sliderNumSamples">
       <property name="geometry">
        <rect>
         <x>114</x>
         <y>198</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>20</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="sliderPosition">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="textExpName">
       <property name="geometry">
        <rect>
         <x>114</x>
         <y>23</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>i.e. &quot;Staph Test&quot;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonAutoReference">
       <property name="geometry">
        <rect>
         <x>138</x>
         <y>360</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto-Set
Reference</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonManualReference">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Manually Set
Reference</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>510</y>
         <width>231</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Reference (Ix)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Reference (Iy)</string>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>410</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>540</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Time Remaining:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="textRefX">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>300</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="textRefY">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>330</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>570</y>
         <width>191</width>
         <height>171</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>system_192x192.png</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonStartContinuous">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>440</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Continuous
Experiment</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridGraphs">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1266</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuLED">
     <property name="title">
      <string>LED...</string>
     </property>
     <addaction name="actionLEDOn"/>
     <addaction name="actionLEDOff"/>
    </widget>
    <widget class="QMenu" name="menuHeating">
     <property name="title">
      <string>Heating...</string>
     </property>
     <addaction name="actionHeatingOn"/>
     <addaction name="actionHeatingOff"/>
    </widget>
    <addaction name="actionReset_MCU_Clock"/>
    <addaction name="menuHeating"/>
    <addaction name="menuLED"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReset_MCU_Clock">
   <property name="text">
    <string>Reset MCU Clock</string>
   </property>
  </action>
  <action name="actionTurn_LED_On">
   <property name="text">
    <string>Turn LED On</string>
   </property>
  </action>
  <action name="actionTurn_LED_Off">
   <property name="text">
    <string>Turn LED Off</string>
   </property>
  </action>
  <action name="actionStop_Heating">
   <property name="text">
    <string>Stop Heating</string>
   </property>
  </action>
  <action name="actionLEDOn">
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="actionLEDOff">
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionHeatingOn">
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="actionHeatingOff">
   <property name="text">
    <string>Off</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderInterval</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelIntervalMinutes</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelDurationSeconds</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSampleRate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelSampleRateMillis</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderNumSamples</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelNumSamples</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
